tests/onnx/quantization/test_ignore_post_processing_nodes.py

```diff
@@ -0,0 +1,205 @@
+        self.reference_ignored_scopes = ['Identity_2', 'Identity_1']
+
+
+class ModelToTest3(NNCFGraphToTest):
+    #          Input_1
+    #             |
+    #           Conv_1
+    #             |
+    #           Identity_1
+    #             |
+    #            TopK_1
+    #             |
+    #           Identity_2
+    #             |      \
+    #            NMS_1   Conv_2
+    #             |        |
+    #          Identity_3 Output_2
+    #             |
+    #           Output_1
+
+    def __init__(self):
+        node_names = ['Input_1', 'Conv_1', 'Identity_1', 'TopK_1', 'Identity_2', 'NMS_1', 'Identity_3', 'Output_1',
+                      'Conv_2', 'Output_2']
+        node_op_types = ['Input', 'Conv', 'Identity', 'TopK', 'Identity', 'NonMaxSuppression', 'Identity', 'Output',
+                         'Conv',
+                         'Output']
+        node_edges = {'Input_1': ['Conv_1'], 'Conv_1': ['Identity_1'], 'Identity_1': ['TopK_1'],
+                      'TopK_1': ['Identity_2'],
+                      'Identity_2': ['NMS_1', 'Conv_2'], 'NMS_1': ['Identity_3'], 'Identity_3': ['Output_1'],
+                      'Conv_2': ['Output_2']}
+        super().__init__(node_names, node_op_types, node_edges)
+        self.reference_ignored_scopes = ['Identity_3']
+
+
+class ModelToTest4(NNCFGraphToTest):
+    #          Input_1
+    #             |
+    #           Conv_1
+    #             |
+    #           Identity_1
+    #             |       \
+    #            TopK_1   Identity_4
+    #             |      /       |
+    #           Identity_2    Identity_5
+    #                \       /
+    #                 \     /
+    #                  NMS_1
+    #                    |
+    #                 Identity_3
+    #                    |
+    #                  Output_1
+
+    def __init__(self):
+        node_names = ['Input_1', 'Conv_1', 'Identity_1', 'TopK_1', 'Identity_2', 'NMS_1', 'Identity_3', 'Output_1',
+                      'Identity_4', 'Identity_5']
+        node_op_types = ['Input', 'Conv', 'Identity', 'TopK', 'Identity', 'NonMaxSuppression', 'Identity', 'Output',
+                         'Identity',
+                         'Identity']
+        node_edges = {'Input_1': ['Conv_1'], 'Conv_1': ['Identity_1'], 'Identity_1': ['TopK_1', 'Identity_4'],
+                      'TopK_1': ['Identity_2'],
+                      'Identity_2': ['NMS_1'], 'NMS_1': ['Identity_3'], 'Identity_3': ['Output_1'],
+                      'Identity_4': ['Identity_2', 'Identity_5'], 'Identity_5': ['NMS_1']}
+        super().__init__(node_names, node_op_types, node_edges)
+        self.reference_ignored_scopes = ['Identity_3', 'Identity_2', 'Identity_5', 'Identity_4', 'Identity_1']
+
+
+# pylint:disable=protected-access
+
+@pytest.mark.parametrize('model_to_test', [ModelToTest1(), ModelToTest2(), ModelToTest3(), ModelToTest4()])
+def test_add_ignoring_nodes_after_last_weight_node(model_to_test):
```

Comment:
Since you are testing common logic, this test is more suitable to be in the common test code. I've moved the rest of the solver tests to common code already. Please do so for this test as well.